src/Control/Monad/Aff.js

```diff
@@ -96,6 +96,61 @@ exports._forkAff = function (nonCanceler, aff) {
   };
 }
 
+exports._forkAll = function (nonCanceler, foldl, affs) {
+  var voidF = function(){};
+
+  return function(success, error) {
+    var cancelers = foldl(function(acc) {
+      return function(aff) {
+        acc.push(aff(voidF, voidF));
+        return acc;
+      }
+    })([])(affs);
```

Comment:
Hah. I don't love it either :/

I might be able to extract out a helper to share with cancelWith.

> On Feb 22, 2016, at 4:11 AM, Gary Burgess <notifications@github.com> wrote:
> 
> In src/Control/Monad/Aff.js:
> 
> > @@ -96,6 +96,61 @@ exports._forkAff = function (nonCanceler, aff) {
> >    };
> >  }
> >  
> > +exports._forkAll = function (nonCanceler, foldl, affs) {
> > +  var voidF = function(){};
> > +
> > +  return function(success, error) {
> > +    var cancelers = foldl(function(acc) {
> > +      return function(aff) {
> > +        acc.push(aff(voidF, voidF));
> > +        return acc;
> > +      }
> > +    })([])(affs);
> 
> 
> —
> Reply to this email directly or view it on GitHub.
>